FastSurferCNN/utils/parser_defaults.py

```diff
@@ -298,6 +299,25 @@ class SubjectDirectoryConfig:
              "determines the voxel size from the image itself (conforming to the minimum voxel size, or 1 if the "
              "minimum voxel size is above 0.95mm). ",
     ),
+    "orientation": __arg(
+        "--orientation",
+        choices=get_args(OrientationType),
+        type=__orientation,
+        dest="orientation",
+        default="lia",
+        help="Select the target affine format for output, native: input defined by input image, soft_lia (or soft-lia, "
```

Comment:
not happy with soft_lia ( soft-lia , soft_lia ) . we should stick to one of them I think.

Reply:
Lets discuss this. I am most happy wioth just `soft_lia` I think, but we can use any of the different versions (right now `soft_lia`, `soft lia` and `soft-lia` are all the same.